src/unique-entities-are-same.ts

```diff
@@ -0,0 +1,12 @@
+import { EntityAttrs } from './types';
+
+export function uniqueEntitiesAreSame(
+	a: EntityAttrs[],
+	b: EntityAttrs[],
+): boolean {
+	const isSameEntity = (a: EntityAttrs, b: EntityAttrs) =>
```

Comment:
Move out of this function (there is no benefit in recreating the function over and over)